### PR TITLE
Add docker-compose

### PR DESCRIPTION
**Pull request by supercid**
## Checklist:
- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] I have assigned the correct milestone or created one if non-existent.
- [ ] I have correctly labeled this pull request.
- [ ] I have linked the corresponding issue in this description.
- [ ] I have updated the corresponding Jira ticket.
- [ ] I have requested a review from at least 2 reviewers
- [ ] I have checked the base branch of this pull request
- [ ] I have checked my code for any possible security vulnerabilities

### CODE REVIEW

File: .docker/scripts/init-and-run.sh [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+set -m
+
+if [ ! -f /var/www/html/magento2/.installed ]; then
+
+  until mysql -u${MYSQL_USER} -p${MYSQL_PASSWORD} -h${MYSQL_HOST}; do
+    >&2 echo "MySQL is unavailable - sleeping"
+    sleep 5
+  done
+  cd /var/www/html/magento2/
+  composer require nosto/php-sdk:@stable
```
Comment (mridang) [inline .docker/scripts/init-and-run.sh:13]: Why is this needed? Where is the actual plugin mounted?

Comment (supercid) [reply .docker/scripts/init-and-run.sh:13]: app/code/Nosto/Tagging.
If we don't have the SDK on Magento, it won't find the references and it won't work

**Comment by supercid** [comment]
I'll close this and move to a standalone repo
